OpenFileIO.md

```diff
@@ -0,0 +1,86 @@
+# "OpenFileIO" Proposal
+
+## Abstract
+
+This proposal calls for the creation of a minimal, open source C library
+based on the prior art of the OpenImageIO `IOProxy`[^1] and Houdini File
+System[^2] APIs. Providing a common cross-platform, cross-application
+extensible abstraction of a random access byte stream. Its purpose is
+to aid the migration away from `fopen` et. al. as the de-facto low-level
+data access mechanism within digital content creation tooling and
+infrastructure. This is required to allow secure data access via other
+means without the need for complex user-space file system emulations or
+elaborate data orchestration techniques.
+
+> N.B. "OpenFileIO" is used colloquially as a place-holder name.
+
+## Problem Statement
+
+It is currently extremely difficult to provide data access to processing
+operations in a media production pipelines though anything other than
+the standard filesystem. This hampers industry adoption of modern
+storage, security and compute architectures.
```

Comment:
I think the problem statement could be a tad clearer. As a reader, i'm not entirely sure what the problem with the current `fopen` to a network share approach is.

An example of what "anything other than the standard filesystem" means, and the capabilities that we currently can't achieve without a more distributed way of accessing data, might help.

Also, due to the "drop in" nature of this solution, is part of the problem that there's already a glut of software that expects a filesystem based approach, despite wanting distributed capability, so the solution must attempt to be an adapter, in order to feasibly allow integration?

Reply:
Does [this](https://github.com/foundrytom/proposals/pull/1/commits/b69223b58c216c5f4cb893ee068e4af7a62d5763) help?